packages/terra-dynamic-grid/src/Region.jsx

```diff
@@ -104,7 +104,13 @@ const Region = ({
   });
 
   const regionClasses = classNames(customProps.className, css(stylesheet.region));
-  return (<div {...customProps} className={regionClasses}>{children}</div>);
+  return (
+    <div {...customProps} className={regionClasses}>
+      <div className="region-child-container">
```

Comment:
this class should be localized via `cx('region-child-container')`

Reply:
Updated https://github.com/cerner/terra-core/pull/1043/commits/1d10ad3a4b1777c2555a6c0bee86be76e4f955f3#diff-e9aa781e0a2995c2193ef9eb6a078ecfR5